src/v2/Components/Inquiry/Logger.ts

```diff
@@ -0,0 +1,44 @@
+import Cookies from "cookies-js"
+import { uniq } from "lodash"
+
+export class Logger {
```

Comment:
So this isn't confused with our normal logger, can this be renamed to `InquiryLogger`?